cloudformation.go

```diff
@@ -102,16 +113,29 @@ func (c *Cloudformation) executeChangeSet(changeSetName string) error {
 
 			return false, nil
 		case cloudformation.StackStatusCreateInProgress, cloudformation.StackStatusUpdateInProgress:
-			duration := time.Duration(6) * time.Second
-			time.Sleep(duration)
+			if time.Since(startOfRetryLoop) > maxWaitTimeForStack {
+				return false, fmt.Errorf("stack still in progress, but maximum wait period of %s has passed", maxWaitTimeForStack)
+			}
 
-			return attempt < 20, errors.New("stack not yet in completed state")
+			time.Sleep(waitFor)
+			waitFor = waitForNext(waitFor)
+			return true, fmt.Errorf("stack still in progress, retrying")
```

Comment:
```suggestion
			waitFor = waitForNext(waitFor)
			
			return true, fmt.Errorf("stack still in progress, retrying")
```

Concept looks good 👍 
I think adding this new line will make the linter pass